### PR TITLE
Adds Ukraine's Independence Day as a holiday, since the child attacking our hosting requested goofball PRs.

### PR DESCRIPTION
**Pull request by Iamgoofball**
The person DDOSing our servers requested all of my PRs be merged, so I figured I'd open a PR for them.

## About The Pull Request

Adds Ukraine's Independence Day as a holiday.

## Why It's Good For The Game

Well, the DDOS person requested we merge goofball PRs, so I'm doing my part to stop the DDOS.
giphy

## Changelog
:cl:
server: Adds Ukraine's Independence Day as a holiday, since the child attacking our hosting requested goofball PRs.
/:cl:

### CODE REVIEW

**Comment by Kubisopplay** [comment]
Give this guy some GBP for this

File: code/modules/holiday/holidays.dm [hunk 1]
```diff
@@ -414,6 +414,16 @@
 
 /datum/holiday/indigenous/getStationPrefix()
 	return pick("Endangered language", "Word", "Language", "Language revitalization", "Potato", "Corn")
+	
+// AUGUST
```
Comment (Mothblocks) [inline code/modules/holiday/holidays.dm:418]: double adding august? fail...

Comment (optimumtact) [reply code/modules/holiday/holidays.dm:418]: dude damn

Comment (Iamgoofball) [reply code/modules/holiday/holidays.dm:418]: whoops

**Comment by WarlockD** [comment]
Think somone could sprite up a clown putin for this day?  "Shoot a clown, save a life?"

**Comment by Lizzzard26** [comment]
shitting at the code, photo in a colors...

**Comment by Mothblocks** [comment]
what

**Comment by optimumtact** [comment]
please stop posting on old pr's

**Pull request merged**